assets/src/design-system/components/contextMenu/index.js

```diff
@@ -13,199 +13,5 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-/**
- * External dependencies
- */
-import PropTypes from 'prop-types';
-import { useCallback, useMemo, useRef, useState } from 'react';
-import styled, { css } from 'styled-components';
-import { v4 as uuidv4 } from 'uuid';
-/**
- * Internal dependencies
- */
-import { BUTTON_TRANSITION_TIMING } from '../button/constants';
-import { useKeyDownEffect } from '../keyboard';
-import { KEYS } from '../../utils/constants';
-import { MenuItem, MenuItemProps } from './menu-item';
-
-const FOCUSABLE_ELEMENTS = ['A', 'BUTTON'];
-
-const SEPARATOR_TOP_CLASS = 'separatorTop';
-const SEPARATOR_BOTTOM_CLASS = 'separatorBottom';
-
-const Popover = styled.div(
-  ({ isOpen, theme }) => css`
-    position: absolute;
-    display: none;
-    margin: 5px 0 0;
-    background-color: ${theme.colors.bg.primary};
-    pointer-events: none;
-    border-radius: ${theme.borders.radius.small};
-    border: 1px solid ${theme.colors.border.disable};
-    box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
-    width: 200px;
-
-    ${isOpen &&
-    css`
-      display: block;
-      z-index: 10;
-      opacity: 1;
-      pointer-events: auto;
-    `};
-  `
-);
```

Comment:
This was split over three components:
|`assets/src/design-system/components/contextMenu/menu.js`|`assets/src/design-system/components/contextMenu/styled.js`|
|--|--|
|![image](https://user-images.githubusercontent.com/22185279/108140946-0790ff80-7080-11eb-889f-51a7aa7bb861.png)|![image](https://user-images.githubusercontent.com/22185279/108140963-124b9480-7080-11eb-882c-836176e83137.png)|